__tests__/meta.test.ts

```diff
@@ -49,7 +58,8 @@ const tagsLabelsTest = async (name: string, envFile: string, inputs: Inputs, exV
   process.env = dotenv.parse(fs.readFileSync(path.join(__dirname, 'fixtures', envFile)));
   const toolkit = new Toolkit();
   const repo = await toolkit.github.repoData();
-  const meta = new Meta({...getInputs(), ...inputs}, new Context(), repo);
+  const context = await getFreshWorkflowContext();
+  const meta = new Meta({...getInputs(), ...inputs}, context, repo);
```

Comment:
I don't think and extra func is necessary:

```suggestion
  const meta = new Meta({...getInputs(), ...inputs}, await getContext(ContextSource.workflow), repo);
```

Reply:
We are forced to use `getFreshWorkflowContext` as `Github.context` is loaded on import, so it does not use the `process.env` mocked variables

Reply:
Ah gotcha you need this to reset mocks